### PR TITLE
Update/clarify the WordPress Object Cache Pro documentation

### PR DESCRIPTION
**Pull request by jkudish**
## Summary

Update + clarify the WordPress Object Cache Pro documentation to explain that certain steps of the setup can only be performed on development or multidev environments.

Additionally, add a warning about the health check notice that is expected to pop up

**Release**:
- [ ] When ready

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- ~Redirect `/old-path/` => `/new-path/` (if applicable)~
- ~Include/exclude pages ^ respectively within docs search service provider (if applicable)~
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by stevector** [comment]
@rachelwhitton can you check with @scottbuscemi on whether this change is necessary?

File: source/content/addons/object-cache/howto/wordpress.md [hunk 1]
```diff
@@ -315,6 +315,8 @@ Refer to the [official Object Cache Pro documentation](https://objectcache.pro/d
       1. Merge code
       1. Activate OCP
       1. Flush Redis cache
+- The `object-cache.php` drop-in file must be created in your development or multidev environment and committed or pushed to live to work.
+- It is expected to see a warning in the WordPress Site Health Status page on your test and live environments saying that Object Cache Pro is unable to access the local filesystem and cannot manage the object cache drop-in. As long as you set it up as per the steps outlined on this page, it is working as expected.
```
[the remark this answers is not in the document]
Comment (jazzsequence) [reply source/content/addons/object-cache/howto/wordpress.md:319]: I don't think this is still accurate. Testing to confirm.

[the remark this answers is not in the document]
Comment (jazzsequence) [reply source/content/addons/object-cache/howto/wordpress.md:319]: Confirmed this no longer shows up in Site Health.
```suggestion
```

File: source/content/addons/object-cache/howto/wordpress.md [hunk 2]
```diff
@@ -3,15 +3,15 @@ title: Enable Object Cache Pro for WordPress
 description: How to install and configure Object Cache Pro for WordPress.
 permalink: docs/object-cache/wordpress
 tags: [cache, plugins, modules, database]
-reviewed: "2023-08-17"
+reviewed: "2024-01-10"
```
Comment (jazzsequence) [inline source/content/addons/object-cache/howto/wordpress.md:6]:
```suggestion
reviewed: "2024-08-08"
```

**Review by jazzsequence — APPROVED** [review]

**Pull request merged**